Bicycle Shop

DanyloSemenov opened the pull request:
[DEMO LINK](https://DanyloSemenov.github.io/layout_miami/

mbruhler reviewed (changes requested):
Hi,

I don't know what is it - maybe a test PR - but for now your layout is completely off, please refer to the mockup in order to fix it

Good luck!

mvjl000 reviewed (changes requested):
1.  Take the streets text on the hero section is way too small.
2. Open menu button on the hero page should be a 'burger'. Only when the menu is opened it should turn into X.
<img width="653" alt="Screenshot 2023-10-23 at 08 26 39" src="https://github.com/mate-academy/layout_miami/assets/65824623/b3860f01-883f-4cb3-956f-932de2767d48">
3. There should be a phone number visible on phone icon hover
4. The mybike logo is to small
5. A phone number in the menu should be 'call-able' a link (`href="tel:"`)
6. Your menu needs more padding at the bottom
7. On a bigger resolutions those should be in the same row
<img width="868" alt="Screenshot 2023-10-23 at 08 33 58" src="https://github.com/mate-academy/layout_miami/assets/65824623/7b7c3e25-5340-47ba-984b-7ba4a56104a0">
Basically you need to copmare your solution to figma very closely, adjust it and you are gonna be on the right track!

DanyloSemenov commented:
I'm so sorry I didn't make commits, I'll correct myself :)

JoyCoffeeAddict reviewed (changes requested):
1. icons have wrong size
![image](https://github.com/mate-academy/layout_miami/assets/131690818/3e21e3b5-ee3d-4599-80bb-6232c7fffe69)
1. image here has too much high
![image](https://github.com/mate-academy/layout_miami/assets/131690818/45c14b89-31ef-40a5-b452-8be49c786e25)
1. Also image above is in low resolution, please use the script that Paweł showed on today's lecture.
1. Email in address is not working, take a look [here](https://css-tricks.com/snippets/html/mailto-links/)
1. Close button in the menu should be exactly in the same spot as open button
![Animation](https://github.com/mate-academy/layout_miami/assets/131690818/4bdee992-982b-49be-bf09-612f012cf185)
1. Links in the menu are not working


Good job overall

DanyloSemenov commented:
and again im remember to make a commits, sorry.
1. corrected icon size
2. added a script for images
3. added email adress 
4. corrected button in the menu
5. corrected links in the menu

mvjl000 reviewed (changes requested):
1. The hero section shouldn't take more than 100vh,
2. Buttons are missing a hover effect (btw. the send button has some default border),
3. Clicking on the logo gives me 404, it shouldn't be that way (checklist points 5, 6)

mbruhler reviewed (changes requested):
Looks great by now! 
One thing I can see is the last bottom photo really blurry. Please fix it.

mbruhler reviewed (changes requested):
I've found one more issue: 
Other images are also blurry. Please check if you use proper size for the viewport
![image](https://github.com/mate-academy/layout_miami/assets/21124163/114aff6d-01ce-440e-99bc-7ec1817512f4)
![image](https://github.com/mate-academy/layout_miami/assets/21124163/e1894077-1213-4f82-906a-c8d2640df87f)

but check also other images

mvjl000 reviewed (changes requested):
In the figma (on mobile), we can see the entire front wheel. You should reposition it a bit to the left. Also the resolution of the photo is not ideal.
![Screenshot 2023-10-26 at 09 25 00](https://github.com/mate-academy/layout_miami/assets/65824623/b7b54a82-aea7-4940-8e31-944ef8a97deb)

JoyCoffeeAddict reviewed (changes requested):
1. Margins and padding on mobile here are a little bit off, so if you open the page on the smallest phone, you can't see the whole menu 
![image](https://github.com/mate-academy/layout_miami/assets/131690818/0c5e64e7-4d26-4c96-975e-d40f896db625)

I think it's the last issue.
